Makefile

```diff
@@ -1,8 +1,11 @@
 CAULKING_VERSION=1.3.1 2022-01-16
```

Comment:
VERSION = 2.0? This is a major change, though technically not breaking any api guarantees.

Reply:
Maybe version 1.99. Next version I'd like to use 

```
SKIP=gitleaks git commit ...
```

as way to signal the pre-commit to skip running gitleaks (with a confirmation by the user).

Reply:
... and use `assert_` throughout..